src/driver/driver_api.cc

```diff
@@ -109,6 +109,51 @@ void GetBinds(const Array<te::Tensor>& args, bool compact,
   }
 }
 
+void GetBinds(const Array<ObjectRef>& args, bool compact,
```

Comment:
Originally I did this because I was making the signature as similar as possible to the other GetBinds (which was originally in C++), which takes in an `Array<te::Tensor>` for the args instead of an` Array<ObjectRef>. 
I'm actually not sure if I should try to get rid of one of the versions of GetBinds, keep them as similar as possible, or just make two versions with different signatures, what do you think?

Reply:
Oh, I didn't notice that other GetBinds. The one you've added here handles a superset of input types, right? In that case my vote would be to remove the old one for the sake of common code. Else, you know someone will end up updating only one of them...

Reply:
Array<te::Tensor> is not a subtype of Array<ObjectRef>, so C++ can't upcast the args and just call the other one. So I won't remove the signature, but I'll have the one that has Array<te::Tensor> as args upcast each element ObjectRef, then call the other GetBinds